config/webpack.prod.config.js

```diff
@@ -15,12 +15,15 @@ const CssNano = require('cssnano');
 const commonConfig = require('./webpack.common.config.js');
 const presets = require('../lib/presets');
 
+const BASENAME = process.env.BASENAME || '/';
```

Comment:
@davidjoy I just realized that this is not relying on dotenv.

Should I add something like this? 
```
dotenv.config({
  path: path.resolve(process.cwd(), '.env'),
});
```

Or is it ok if this config is not configurable through dotenv and instead through environmental variables? If that is the case, I think  that I can apply this change to the webpack.common.config.js instead

Reply:
Any thoughts? @davidjoy

Reply:
@morenol I think putting this in webpack.common.config.js and reading through environment variables makes sense.